packages/xdl/src/logs/PackagerLogsStream.ts

```diff
@@ -396,7 +393,7 @@ export default class PackagerLogsStream {
       if (bundleComplete) {
         if (this._onFinishBuildBundle) {
           const error = msg.type === 'bundle_build_failed' ? 'Build failed' : null;
-          this._onFinishBuildBundle(error, this._bundleBuildStart, bundleBuildEnd, progressChunk);
+          this._onFinishBuildBundle(error, this._bundleBuildStart || new Date(0), new Date(), progressChunk);
```

Comment:
Not sure about `new Date(0)`... it's 1st of January 1970. This might cause some funny errors if we ever calculate the duration from those dates and this happens. "Build took 49 years ..." etc.